spec/models/selected_assistance_expert_spec.rb

```diff
@@ -54,5 +64,85 @@
 
       it { is_expected.to eq [selected_assistance_expert] }
     end
+
+    describe 'with_status' do
+      let!(:selected_ae_with_status_quo) { create :selected_assistance_expert, status: :quo }
+      let!(:selected_ae_taken_care_of) { create :selected_assistance_expert, status: :taking_care }
+      let!(:selected_ae_with_status_done) { create :selected_assistance_expert, status: :done }
+      let!(:selected_ae_not_for_expert) { create :selected_assistance_expert, status: :not_for_me }
+
+      it do
+        expect(SelectedAssistanceExpert.with_status(:quo)).to eq [selected_ae_with_status_quo]
+        expect(SelectedAssistanceExpert.with_status(:taking_care)).to eq [selected_ae_taken_care_of]
+        expect(SelectedAssistanceExpert.with_status(:done)).to eq [selected_ae_with_status_done]
+        expect(SelectedAssistanceExpert.with_status(:not_for_me)).to eq [selected_ae_not_for_expert]
+      end
+    end
+
+    describe 'updated_before_five_days_ago' do
+      subject { SelectedAssistanceExpert.updated_before_five_days_ago }
+
+      let!(:selected_ae_updated_two_weeks_ago) { create :selected_assistance_expert, updated_at: 2.weeks.ago }
+
+      before { create :selected_assistance_expert, updated_at: 4.days.ago }
+
```

Comment:
why the before and the let! ? this seems to be a bit inconsistent

Reply:
You need to use `let!(:foo) { bar }` when you need to reuse `foo` elsewhere